docs/sources/http_api/team.md

```diff
@@ -134,6 +134,8 @@ Status Codes:
 
 The Team `name` needs to be unique. `name` is required and `email`,`orgId` is optional.
 
+Optionally, users can supply also an `id`; in that case, the new team is created with the provided identifierr instead of the autoincremented one assigned by the database backend.
```

Comment:
```suggestion
Optionally, users can supply also an `id`; in that case, the new team is created with the provided identifier instead of the autoincremented one assigned by the database backend.
```